src/main/java/com/cloudant/client/org/lightcouch/CouchDbProperties.java

```diff
@@ -229,4 +251,39 @@ public CouchDbProperties setAuthenticatedModeSSLSocketFactory(SSLSocketFactory f
         this.authenticatedModeSSLSocketFactory = factory;
         return this;
     }
+
+    public Map<String, String> getCustomHeaders() {
```

Comment:
custom headers don't need to be supported here (we'll just use interceptors if we need a custom header).